sockets/client-tcp.c

```diff
@@ -48,6 +48,7 @@ int main(int argc, char ** argv)
                         break;
                 }
         }
+        message = "gr8 b8 m8, r8 8/8";
```

Comment:
part of our goal here is to discover the horror of shared repos with merging/conflicts, etc. Have fun!